### DISCUSSION TITLE
Idea for mirrors

### DISCUSSION THREAD

**Discussion by MIfoodie** [SHSGames/shsgames.github.io#253] [General]
I would like a requirement for mirrors on the mirror list to be that they must be different. It just doesn't make sense that there are exact copies of shsgames.github.io.

**Comment by JoshMerlino** [comment]
Mirrors are used so if schools where to block the site, there would be many other links that will deliver the exact same experience making it extremally difficult for ISPs/school districts to cut off access to SHSGames.

**Reply by hello-smile6** [reply]
https://github.com/SHSGames/shsgames.github.io/issues/399#issuecomment-1107348090
